### DISCUSSION TITLE
Can you import database api to ckan

### DISCUSSION THREAD

**Discussion by kyung064** [ckan/ckan#6946] [Q&A]
hi i have an old data portal which actually works quite perfectly, but since we want to use ckan can we intergrated the old database api with ckan? if can may i ask for sollution bcs i cant find one (pretty newbie) thanks

**Comment by amercader** [comment]
If the old data portal is not a CKAN instance you will need to create a migration script which translates the old API to the one expected by CKAN to create datasets, etc Here are the API docs to get you started: http://docs.ckan.org/en/latest/api/index.html#
